src/main/scala/nl.knaw.dans.easy.solr4files/Command.scala

```diff
@@ -43,7 +42,16 @@ object Command extends App with DebugEnhancedLogging {
     .doIfFailure { case e => logger.error(e.getMessage, e) }
     .doIfFailure { case NonFatal(e) => println(s"FAILED: ${ e.getMessage }") }
 
+
   private def runCommand(app: EasyUpdateSolr4filesIndexApp): Try[FeedBackMessage] = {
-    Success("")
+    commandLine.subcommand
+      .collect {
+        case update @ commandLine.update => app.update(update.bagStore(), update.bagUuid())
+        case delete @ commandLine.delete => app.delete(delete.bagUuid())
+        case init @ commandLine.init if init.bagStore.isSupplied => app.initSingleStore(init.bagStore())
+        case init @ commandLine.init if !init.bagStore.isSupplied => app.initAllStores()
```

Comment:
you don't need the extra `!init.bagStore.isSupplied` here (can you figure out why?).

alternatively you could do

```scala
case init @ commandLine.init => init.bagStore.toOption
    .map(app.initSingleStore)
    .getOrElse(app.initAllStores())
```